setup.py

```diff
@@ -34,6 +34,7 @@
 
 WITH_DISTRIBUTED = not check_env_flag('NO_DISTRIBUTED') and not IS_WINDOWS
 WITH_DISTRIBUTED_MW = WITH_DISTRIBUTED and check_env_flag('WITH_DISTRIBUTED_MW')
+WITH_GLOO_IBVERBS = check_env_flag('WITH_GLOO_IBVERBS')
```

Comment:
Is there any command-line utility we could use to try to autodetect IBVerbs (e.g. `ibstat`)? It would be nicer to add some helpers in `tools.setup_helpers` that try to run a few commands and if any of those exist we turn this flag on automatically.

Reply:
This seems hacky since we can't expect all users to have ibstat installed on their machine.  Let's do this then, we always build Gloo with both TCP and IB support, so that we don't need this flag. (Or we always set this flag on).

The purpose of autodetect and set this flag on is that if the machine has IB, we build/use the IB, and if the machine doesn't, we build/use the TCP. Why not we just build Gloo for both? And then in Gloo data channel, use the API to detect if we have IB interfaces. If yes, use IB, otherwise, fallback to TCP.